packages/components/avatar/src/avatar.ts

```diff
@@ -49,15 +52,27 @@ export class Avatar extends ScopedElementsMixin(LitElement) {
   /** The slotted badge element. */
   badge?: Badge;
 
-  /** The clip-path cutout for the badge. */
+  /** @internal The clip-path cutout for the badge. */
   @state() clipPath?: string;
 
+  /**
+   * The color of the avatar.
+   * @default grey
+   */
+  @property({ reflect: true }) color?: AvatarColor;
```

Comment:
This is probably not the right place, but it looks like colors are not described on the documentation website (in the `usage` tab)?
 https://salmon-hill-037b61403-2577.westeurope.6.azurestaticapps.net/categories/components/avatar/usage#figma-options
 
<img width="1155" height="1456" alt="image" src="https://github.com/user-attachments/assets/b30cc5f3-e1e6-4aa1-86db-4439ddb543d3" />

Reply:
And probably `badge` 'Figma options' are not updated as well?

https://salmon-hill-037b61403-2577.westeurope.6.azurestaticapps.net/categories/components/badge/usage